tests/test_data.py

```diff
@@ -248,7 +248,8 @@ def test_data_parse_state_json(self):
 """
 
     def test_data_parse_config_xml(self):
-        dnode = self.ctx.parse_data_mem(self.XML_CONFIG, "xml", validate_present=True)
+        dnode = self.ctx.parse_data_mem(self.XML_CONFIG, "xml", validate_present=True,
+                                        validate_multi_error=True)
         self.assertIsInstance(dnode, DContainer)
```

Comment:
Could you add a specific unit test for your new option instead, that would verify that we get those multiple errors?

Reply:
@samuel-gauthier I got it, I'll add the test, thanks